demosplan/DemosPlanCoreBundle/Resources/client/scss/components/_data-table.scss

```diff
@@ -158,4 +158,23 @@
 
         background-color: $dp-color-white;
     }
+
+    .table-text {
+        & ul,
+        & ol {
+            padding-left: 1.3em;
+
+            & li > p {
+                margin: 0;
+            }
+        }
```

Comment:
I am not sure, but if I did't forget everything about scss we can omit the `&` all along.